tests/e2e_test/test_migration.py

```diff
@@ -66,14 +66,14 @@ def parse_manager_log_file(log_file):
 @pytest.mark.asyncio
 @pytest.mark.skipif(torch.cuda.device_count() < 2, reason="at least 2 gpus required for migration bench")
 @pytest.mark.parametrize("model", ['/mnt/model/Qwen-7B'])
-@pytest.mark.parametrize("migration_backend", ['rpc', 'gloo', 'nccl'])
-@pytest.mark.parametrize("enable_pd_disagg", [False, True])
-async def test_migration_benchmark(model, migration_backend, enable_pd_disagg):
+@pytest.mark.parametrize("migration_backend", ['rpc', 'gloo'])
+async def test_migration_benchmark(model, migration_backend):
     base_port = 37037
     instance_output_logs = []
 
     device_count = torch.cuda.device_count()
-    num_dispatch_instances = device_count//2 if enable_pd_disagg else math.inf
+    enable_pd_disagg = False
```

Comment:
why not remove enable_pd_disagg here?

Reply:
Get.

remove enable_pd_disagg local var.